pos-frontend/src/components/SingIn.jsx

```diff
@@ -0,0 +1,205 @@
+import { useState } from 'react'
+import { Container, Grid, CssBaseline, Typography, Snackbar, Alert, TextField, Box, Avatar, Button } from '@mui/material'
+import { Link, useNavigate } from "react-router-dom"
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined'
+import { useForm } from "react-hook-form"
+import axios from "axios"
+import { url } from '../config.js'
+
+const initialState = {
+  open: false,
+  status: '',
+  message: ''
+}
+
+const SignIn = (props) => {
+
+  const [ message, setMessage ] = useState(initialState);
+  const { register, getValues, handleSubmit, formState: { errors }, setError, setFocus } = useForm({ mode: 'onTouched'});
+  let navigate =  useNavigate();
+
+  const handleClose = () => {
+    setMessage({
+      open: false,
+      status: '',
+      message: ''
+    })
+  }
+  
+
+  const onSubmit = (event) => {
+    
+    const data = getValues();
+    
+    let updatedUrl = url + "/auth/signin"
```

Comment:
use template literals to concatenate string with any dynamic value

Reply:
Completed